### PR TITLE
Mitigation Timeline Modifications

### PR DESCRIPTION
**Pull request by PruthuviXeptagon**
Set Parent Mandatory for Activity Add.
Set Parent's Start Year to Mitigation timeline Start Year (Also set 30 as Year Range).
Removed Activity Link Unlink From Parent Side.

### CODE REVIEW

File: backend/services/src/validation/payload.validator.ts [hunk 1]
```diff
@@ -32,7 +33,7 @@ export class PayloadValidator {
 		}
 	}
 
-	validateMitigationTimelinePayload(mitigationTimelineDto: mitigationTimelineDto) {
+	validateMitigationTimelinePayload(mitigationTimelineDto: mitigationTimelineDto | ActivityDto, method: string) {
```
Comment (tharindugayanga) [inline backend/services/src/validation/payload.validator.ts:36]: Make the method an enum

File: web/src/Components/Timeline/timeline.tsx [hunk 2]
```diff
@@ -9,6 +9,8 @@ interface Props {
   actualTimeline: any;
   loading: boolean;
   method: 'create' | 'view' | 'update';
+  mtgStartYear: any;
```
Comment (tharindugayanga) [inline web/src/Components/Timeline/timeline.tsx:12]: mtgStartYear and mtgRange can be number type

File: backend/services/src/validation/payload.validator.ts (hunk 1 above)
Comment (PruthuviXeptagon) [reply backend/services/src/validation/payload.validator.ts]: Fixed

File: web/src/Components/Timeline/timeline.tsx (hunk 2 above)
Comment (PruthuviXeptagon) [reply web/src/Components/Timeline/timeline.tsx]: Fixed

**Pull request merged**